src/main/java/com/google/api/generator/gapic/composer/Composer.java

```diff
@@ -191,17 +193,14 @@ public static List<GapicClass> generateTestClasses(GapicContext context) {
 
   @VisibleForTesting
   static List<GapicClass> prepareExecutableSamples(List<GapicClass> clazzes, String protoPackage) {
-    //  parse protoPackage for apiVersion and apiShortName
+    //  parse protoPackage for apiVersion
     String[] pakkage = protoPackage.split("\\.");
     String apiVersion;
-    String apiShortName;
     //  e.g. v1, v2, v1beta1
     if (pakkage[pakkage.length - 1].matches("v[0-9].*")) {
       apiVersion = pakkage[pakkage.length - 1];
-      apiShortName = pakkage[pakkage.length - 2];
     } else {
       apiVersion = "";
-      apiShortName = pakkage[pakkage.length - 1];
     }
     //  Include license header, apiShortName, and apiVersion
     return clazzes.stream()
```

Comment:
Nice to have: I know this is pre-existing code, it would be nice to refactor this part to  be more readable/understandable

- `map` is usually used to map an object to another, here we are not really mapping, we are adding something to the existing object, so we can use `foreach` or just traditional for loop. 
- Nested mappings and lambda expressions make it really hard to read, there are six levels of indentation here, it's usually a code smell that you have more than two levels of indentation.

Reply:
Thanks for the advice - makes sense. I've refactored to use `forEach`.